flow/raster_cache.cc

```diff
@@ -299,35 +299,33 @@ void RasterCache::SetCheckboardCacheImages(bool checkerboard) {
 
 void RasterCache::TraceStatsToTimeline() const {
 #if !FLUTTER_RELEASE
+  FML_TRACE_COUNTER("flutter", "RasterCache", reinterpret_cast<int64_t>(this),
+                    "LayerCount", layer_cache_.size(), "LayerMBytes",
+                    EstimateLayerCacheByteSize() * 1e-6, "PictureCount",
```

Comment:
nit: we might want to consider making a constexpr like `kMegaBytes` or something for this. I'm thinking the same for another PR where I'm dividing by 1,000,000 to the same effect.

Reply:
Can we make it be 2^20 to match the way dart vm calculates mbytes?

Reply:
Sure, changed it to `1 << 20`. Does it look good?